src/Extensions/Compatability/Logger/Category.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using Microsoft.Omex.Extensions.Compatability.Validation;
+
+namespace Microsoft.Omex.Extensions.Compatability.Logger
+{
+	/// <summary>
+	/// Logging category
+	/// </summary>
+	public struct Category : IEquatable<Category>
+	{
+		/// <summary>
+		/// Category
+		/// </summary>
+		/// <param name="name">name</param>
+		/// <exception cref="ArgumentNullException"><paramref name="name"/> is null or white space</exception>
+		public Category(string name)
+		{
+			Name = Code.ExpectsNotNullOrWhiteSpaceArgument(name, nameof(name), null);
+		}
+
+
+		/// <summary>
+		/// Name
+		/// </summary>
+		public string Name { get; }
+
+
+		/// <summary>
+		/// Indicates whether the current object is equal to another object of the same type
+		/// </summary>
+		/// <param name="other">An object to compare with this object</param>
+		public bool Equals(Category other)
+		{
+			if (ReferenceEquals(null, other))
+			{
+				return false;
+			}
+
+			if (ReferenceEquals(this, other))
+			{
+				return true;
+			}
+
+			return string.Equals(Name, other.Name, StringComparison.InvariantCulture);
```

Comment:
>InvariantCulture [](start = 59, length = 16)

This should be `Ordinal` or `OrdinalIgnoreCase`. `InvariantCulture` performs unnecessary operations.

Reply:
Fixed, thanks